pkgs/applications/editors/jetbrains/update_ides.py

```diff
@@ -12,6 +12,8 @@
 updates_url = "https://www.jetbrains.com/updates/updates.xml"
 current_path = pathlib.Path(__file__).parent
 versions_file_path = current_path.joinpath("versions.json").resolve()
+fromVersions = {}
+toVersions = {}
```

Comment:
can you not just use a string here?

Reply:
I guess the min/max calculation would not work with simple strings (see my explanation below).